### ISSUE TITLE
Tabbing out of the inventory

### ISSUE THREAD

**Issue by Hexorism** [iONDegen/ps-inventory#20]
When tabbing out of the inventory, screen becomes blurry cant do anything and have to force close the game.

**Comment by Brave-developments** [comment]
did you rename the folder?

**Comment by ScrewWorldRP** [comment]
I'm have the same issue and solutions? @Brave-developments

**Comment by Brave-developments** [comment]
did you rename the folder?

**Comment by ScrewWorldRP** [comment]
> did you rename the folder?

Yeah I sat with it a for a while and after a lot of failed attempts, it dawned on me that I needed to replace all of the ps-inventory with qb-inventory since I changed the original name of the folder. Shortly after I stumbled across your video which would have saved me hrs of trial and errors lol. Thank for the script it’s awesome! I only have one other issue with the backpack section overlap on 1440p if you could help it would be amazing @Brave-developments.

**Comment by Brave-developments** [comment]
thats something to be addressed
